doc/usecases/elektra_web/UC_duplicating_keys.md

```diff
@@ -0,0 +1,20 @@
+# Use Case: Duplicating keys
```

Comment:
@markus2330 I was going to add an icon to duplicate keys, because this will be a pretty common use case (e.g. creating a new student with the same fields)